### PR TITLE
Remove the networked image from the example for `FittedBox`

### PR DESCRIPTION
**Pull request by gspencergoog**
## Description

This removes the `Image.network` from the example, since that causes a race condition in the test, and an image isn't necessary for the example, it works just was well (maybe better) with a `Placeholder` widget that shows how the fitted box affects the layout.

## Related
- Caused a revert of the Dart roll because of the racy test: https://github.com/flutter/engine/pull/40767

## Tests
 - No change to the tests because the example test doesn't cover the content of the widget drawn.

### CODE REVIEW

**Review by jason-simmons — APPROVED** [review]

**Comment by zanderso** [comment]
@CaseyHillers here's another instance of g3 testing being blocked on a failing but ignored target.

Landing manually to unblock the Dart -> Engine roll.

**Pull request merged**
